react-hs/src/React/Flux/Internal.hs

```diff
@@ -100,13 +104,17 @@ instance Show HandlerArg where
 -- | Marker class to make things less polymorphic (we likely only ever need 'ViewEventHandler',
 -- 'StatefulViewEventHandler', so just throwing around unconstrained type variables is a little
 -- confusing).
-class IsEventHandler handler
+data EHandler s = EHView | EHState s
```

Comment:
do we need the `s` parameter here?  isn't it always `*`, so we can keep it implicit?